### ISSUE TITLE
Fixes needed for problems with navigation and using the game.

### ISSUE THREAD

**Issue by jordanmack** [Magickbase/nervos-official-website#430]
Some users have had issues with page navigation and with the game. In the interest of giving the best user experience while realizing we have limited resources available, we need to prioritize functionality over cosmetics and nice-to-have features.

Per discussion with Matt, I am submitting the following recommendations:

1. Update key bindings to prioritize navigation.
    - Up and Down should be unbound from the game of life animation.
    - Bind the Home and End keys to scroll to the top and bottom.
    - Page Up, Page Down, Up, and Down should move through the pages.
2. Simplify the game.
    - Unbind right-click from drawing. Bind left-click to draw on the desktop.
    - Remove the ability to move the animation via dragging or arrow keys.
    - Disable the automatic timed on/off for animation playback. (Always start on and stay on.)
    - Remove the zoom buttons from the control button bar.
    - Enable edge wrapping if time permits.
3. Simplify functionality on touch devices.
    - Remove right-bottom control buttons entirely from touch devices.
    - Remove the ability to move the animation via dragging (also noted above).
4. Fix cosmetic issues.
    - Fix the animation sometimes appearing off-screen on load. If the cause cannot be found disable animation on touch devices and use a static background until a real fix is found.
    - Fix occasional screen blanking/breaking on touch devices. #429 If the cause cannot be found disable animation on touch devices and use a static background until a real fix is found.

A full log of all issues can be found [here](https://docs.google.com/document/d/1QovKGuGvwFxJnEQbuMGvz5JdOK22j5ApBWy45DU-Uuo/edit), which contains more details about anything mentioned above.

**Comment by WhiteMinds** [comment]
> Unbind right-click from drawing. Bind left-click to draw on the desktop.

Do you still want to keep the prompt for "LEFT CLICK TO DRAW"? 

> Enable edge wrapping if time permits.

What specifically does "edge wrapping" refer to?

> Remove right-bottom control buttons entirely from touch devices.

Does this include large-screen mobile devices like iPads?

**Comment by WhiteMinds** [comment]
PR: https://github.com/Magickbase/nervos-official-website/pull/436

> * Update key bindings to prioritize navigation.
>   
>   * Up and Down should be unbound from the game of life animation.
>   * Bind the Home and End keys to scroll to the top and bottom.
>   * Page Up, Page Down, Up, and Down should move through the pages.
> * Simplify the game.

Resolved.

>   * Unbind right-click from drawing. Bind left-click to draw on the desktop.
>   * Remove the ability to move the animation via dragging or arrow keys.
>   * Disable the automatic timed on/off for animation playback. (Always start on and stay on.)
>   * Remove the zoom buttons from the control button bar.
>   * Enable edge wrapping if time permits.
> * Simplify functionality on touch devices.

Everything has been resolved except for `edge wrapping`.

>   * Remove right-bottom control buttons entirely from touch devices.
>   * Remove the ability to move the animation via dragging (also noted above).

I need to wait for a response to the question I commented on above before starting the processing.

> * Fix cosmetic issues.
>   
>   * Fix the animation sometimes appearing off-screen on load. If the cause cannot be found disable animation on touch devices and use a static background until a real fix is found.
>   * Fix occasional screen blanking/breaking on touch devices. [Pages sometimes break and go blank. #429](https://github.com/Magickbase/nervos-official-website/issues/429) If the cause cannot be found disable animation on touch devices and use a static background until a real fix is found.

Unable to reproduce, need help from @Keith-CY or the testing department.

**Comment by jordanmack** [comment]
@WhiteMinds My apologies for not responding sooner. I did not see the message until now.

> Do you still want to keep the prompt for "LEFT CLICK TO DRAW"?

Keep on Chrome desktop only.

> What specifically does "edge wrapping" refer to?

It means when the cells move off the screen they appear on the opposite side. eg: If it moves off the right side of the screen, you see it appear on the left side. If it goes off the bottom, it appears at the top.

> Remove right-bottom control buttons entirely from touch devices.
> Does this include large-screen mobile devices like iPads?

Yes, the buttons are too hard to press on all touch screen devices so remove them as a temporary fix.

**Comment by jordanmack** [comment]
After trying the newest deployed version, I have a few more recommendations:

1. Remove rewind functionality. It doesn't work intuitively. (Only works when game is paused.)
    - Remove the rewind button from the button bar.
    - Unbind backspace from the game.
2.  Prioritize navigation over the game with the space bar.
    - Unbind the space bar from game.
    - Bind the space bar to scroll to the next page. (Next page is traditional web browser functionality.)
3. Unbind zoom keys.
    - Unbind + and - from the game. The current functionality is problematic and doesn't add much value.
4. Update the help page to match current functionality.

Notes:
I regularly have problems with the game key bindings not working. However, it is probably not worth the effort to fix at this point, which is why I'm recommending just removing some of the problematic functions.

Pressing tab has occasionally caused the page to scroll halfway between two sections, effectively breaking it and forcing the user to refresh. However, this seems to be very rare so it can be skipped for the time being.

**Comment by Keith-CY** [comment]
@PainterPuppets could you do me a favor to handle this issue
